addons/account_bank_statement_import/res_partner_bank.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of account_bank_statement_import,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     account_bank_statement_importis free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     account_bank_statement_import is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with account_bank_statement_import_coda.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import re
+from openerp import api, models, fields
+
+
+class ResPartnerBank(models.Model):
+    _inherit = 'res.partner.bank'
+
+    sanitized_acc_number = fields.Char(
```

Comment:
that's a good point, but i wonder if it wouldn't be easier if we just overwrite the create() and write() functions in order to always save the acc_number sanitized. Also, i wonder if it's useful to sanitize right operand of the search... what do you think? Do you have example where it would be definitively needed?